documentation/features/runner.mdx

```diff
@@ -0,0 +1,46 @@
+---
+sidebarTitle: Runner
+title: Runner
+description: Iterate and execute your requests in a collection.
+---
+
+The Runner is a powerful tool that allows you to run a collection of requests sequentially. You can also configure the run settings to suit your needs.
+
+## Running a Collection
+
+To run a collection:
+
+- Right-click on the collection and select the "**Run Collection**" option.
+- Alternatively, hover over the collection name and click on the "**Run Collection**" icon.
+
+You can choose to run the collection from the CLI or through the web interface based on your preference. A new tab will open where you can configure the run settings as described below.
+
+## Run Settings
+
+1. **Delay**: Set an interval delay (in milliseconds) between running each request.
+2. **Stop Run if Error Occurs**: The collection run stops if an error is encountered within a script or if there's a problem sending a request.
+3. **Persist Responses**: Log response headers and bodies for review after running the collection. Note that persisting responses may impact performance for large collections.
+4. **Keep Variable Values**: Persist the variables used in the run, so any variables updated during the run will retain their changes after completion.
```

Comment:
This isn't a supported run configuration in the current implementation.

```suggestion
```

Reply:
Got it, tested the feature out!